docs/INSTALL.md

```diff
@@ -25,31 +17,22 @@ Templates and other build generated files are hosted on an S3 bucket by project
 These templates are ready to use to install the latest release of MSAM.
 
 ```
```

Comment:
The way we are building, I think only the main template ends up in the main solutions-reference bucket. The rest of the templates will be in regional buckets. Might need to review VWR to compare.